cli/src/cli.rs

```diff
@@ -76,16 +77,18 @@ impl SourceLineProvider for CachedSourceLocationLineProvider {
 }
 
 #[must_use = "The pipeline status should be checked"]
-pub fn use_pipeline<'a>(
+#[allow(clippy::too_many_arguments)]
+pub fn use_pipeline(
     entry_point: &Name,
     starting_page: SourceId,
     analyzer: &Analyzer<'_>,
+    externals: &Externals,
     vm: &mut VM,
     diagnostics: Vec<Diagnostic>,
-    importer: &mut (impl ASTImporter<'a> + ErrorReporter),
+    errors: Vec<FileImportError>,
+    sources: &SourcesCache,
```

Comment:
Pass the importer directly, as it already contains those two things.

Reply:
no it does not contains `sources` anymore